src/service-worker.js

```diff
@@ -1,6 +1,12 @@
 /* eslint-disable no-undef */
 // eslint-disable-next-line no-restricted-globals, no-underscore-dangle
-workbox.precaching.precacheAndRoute(self.__precacheManifest || []);
+const precacheManifest = self.__precacheManifest || [];
+
+workbox.precaching.precacheAndRoute(
+  precacheManifest.filter(
+    (p) => !p.url.includes("graphpage") && !p.url.includes("legacy")
```

Comment:
This is so that the service worker doesn't ask the browser to precache these pages.